packages/rich-text/src/plugins/List/index.tsx

```diff
@@ -85,8 +85,6 @@ export const withListOptions: CustomSlatePluginOptions = {
   // ELEMENT_LIC is a child of li, slatejs does ul > li > lic + ul
   [ELEMENT_LIC]: {
     type: BLOCKS.PARAGRAPH,
-    // TODO: We might want to change it to our paragraph in case we develop one, otherwise we can simply use the slatejs default element
-    component: Slate.DefaultElement,
```

Comment:
The `component` property is ignored because of this: https://github.com/contentful/field-editors/pull/753/files#diff-f887640d5b7f73e4a4d28d39eeea93db01913e49a8a3c1ea12ab7349af480fb8R40